views/website/libraries/9-Go.json

```diff
@@ -5,13 +5,13 @@
   "bgColor": "rgb(71, 182, 196)",
   "libs": [
     {
-      "minimumVersion": null,
+      "minimumVersion": "1.15",
```

Comment:
The minimum version reflects the minimum version of the library that has support for all things marked true, not the language version. I think this should be `v4.0` here since you just released it :)

Reply:
Ah, understood, in that case is v3.2.2 because this added eddsa support and support for `aud` was already there for a while.